content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+    *   [More information on password security from the Finnish National Cyber Security Centre](https://www.kyberturvallisuuskeskus.fi/en/ncsc-news/instructions-and-guides/longer-better-how-create-strong-password)
+*   Start using two-factor authentication on all services, such as email or social media sites. Two-factor authentication is usually mandatory in banking services. Two-factor authentication confirms your identity using two separate methods. While SMS is possible, a special authentication app is recommended, such as Google Authenticator ([Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2),[iOS](https://apps.apple.com/fi/app/google-authenticator/id388497605)) or [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator).
+
+### Check your insurance policy
+
+*   Many home insurances provide limited legal assistance for victims of an identity fraud. The data privacy legislation may give you a right for damage compensation.
+
+### Make a request to access your personal data from Vastaamo
+
+*   According to European General Data Protection Regulation (GDPR), you have the right to know what data Vastaamo has collected about you. This can help you decide what you want to do next.
+*   Contact Vastaamo with their [request for access to personal data](https://vastaamo.fi/files/rekisteritietojen_tarkastuspyynto.pdf) -form. More information about Vastaamo Data Protection is available from [their website](https://vastaamo.fi/tietosuoja/). (currently only in Finnish)
+*   Understand that it may take several weeks to get an answer.
+
+### Request a prohibition of direct marketing from Traficom and Data & Marketing Association of Finland (ASML)
+
+*   A consumer can limit phone marketing by calling Data & Marketing Association of Finland (ASML) service number 0600 13404 from Monday to Friday between 8 and 18. The service costs 0,39€ / minute + operator service charges. Queuing is free-of-charge. (Note. You can't call from abroad.)
+*   ASML does not charge any other fees from the caller. More information available from [ASML website](https://www.asml.fi/kieltopalvelut/). (currently only in Finnish)
+*   Prohibition of data disclosure from Finnish Transport and Communications Agency Traficom registries can be made from [Traficom website](https://www.traficom.fi/en/services/how-prohibit-disclosure-your-data).
+
+### Your Right to be Forgotten from search results
+
+*   The GDPR grants you the right to request removal of certain personal information from Internet search engines.
+*   [Google Personal Information Removal Request Form](https://www.google.com/webmasters/tools/legal-removal-request?complaint_type=rtbf&hl=en&rd=1)
+*   [Bing Request to Block Bing Search Results In Europe](https://www.bing.com/webmaster/tools/eu-privacy-request)
+
+
+### Take care of yourself
+
+*   Don't panic. It's possible that only some of your data has been leaked.
```

Comment:
I would move this to the top.

Don't Panic
While it is possible that some of your data may have been leaked, there is support and advice available.

Reply:
This is not a bad idea. I would suggest that we do it also for the Finnish version if we do that.

Reply:
It has been moved up in both versions now.